api/src/test/java/io/druid/data/input/impl/TimestampSpecTest.java

```diff
@@ -37,6 +39,29 @@ public void testExtractTimestamp() throws Exception
   }
 
   @Test
+  public void testExtractTimestampWithTimezone() throws Exception
+  {
+    final DateTimeZone saved = DateTimeZone.getDefault();
+    final DateTimeZone dataTimezone = DateTimeZone.forID("Asia/Shanghai");
+    try {
+      DateTimeZone.setDefault(DateTimeZone.UTC);
```

Comment:
is it necessary to set timezone for this test?
It should be enough to test that the resulting extracted timestamp has the requested timezone set, no?